code/computational_geometry/src/cohen_sutherland_lineclip/cohenSutherlandAlgorithm.cpp

```diff
@@ -0,0 +1,156 @@
+#include <iostream>
```

Comment:
Please follow our C++ style guide located [here](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c%2B%2B).

Reply:
I have updated changes which you have suggested please, review it.

Reply:
I have updated changes which you have suggested please, review it.